public/Algorithm/BinaryTree/claude sonnet 4.6 extended/105. Construct Binary Tree from Preorder and Inorder Traversal/README_React.html

```
@@ -0,0 +1,1087 @@
+    d.addEventListener("toggle", () => {
+      const arrow = d.querySelector(".arrow");
+      if (arrow) arrow.style.transform = d.open ? "rotate(90deg)" : "rotate(0deg)";
+    });
+  });
+</script>
+</body>
+</html>
+g)";
+    });
+  });
+</script>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </script> ]

(tag-pair)